### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by adeola003**
To implement the changes required by the new project I :
- created a new js file;
- I used a selector to create variables linked to my menu, its elements, and the hamburger icon;
- I changed the behavior of the menu when the user clicks on the hamburger button and pushed the changes;
- I added my partner as a collaborator and he reviewed my code and made some suggestions then added linters to the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
    <header>
     <nav class="navbar">
         <div class="hamburger">
-            <img src="./assets/ic_menu.svg" alt="hamburger">
+            <img class="hamb" src="./assets/ic_menu.svg" alt="hamburger">
         </div>
```
Comment (TadesseDev) [inline index.html:20]: - [ ] Good job @adeola003, Please here notice that the icon is almost hidden on a `375px` display and I have to scroll to see the whole icon. Please make sure to add some padding and margin as visible on the figma design so the user has a better experience using your application. 

### Your design 
image

### Figma design 
image

File: styles.css [hunk 2]
```diff
@@ -31,17 +31,46 @@ body {
...
+    color: #000;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 35px;
+    filter: blur(1px);
     width: 100%;
+    left: 0;
+    height: 610px;
   }
```
Comment (TadesseDev) [inline styles.css]: - [ ] for your menu items, Please make sure the list items are the same as the ones on the figma design. this includes `font-family`, `font-size`, and `text-decoration`/remove the underline, and please no blue effect on the text.
- [ ] Also, when the menu opens, I notice the close/`X` icon is not fully visible. Please address this to have a better layout and provide a better user experience. 

### Your design 
 
image

### Figma design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -31,17 +31,50 @@ body {
     background-color: #e5e5e5;
   }
 
-  .menu {
+  .sections {
```
Comment (codecaiine) [inline styles.css:34]: - [ ] I can see a few differences between your design and Figma design,  As mentioned by the previous reviewer it is very important to stick to the Figma design as much as possible in order to have a beautiful website and good user experience 👍. As we can see, your Xmark design and menu option font size and weight, spacing(padding/margin), and position are a bit different, please check the screenshots below ⬇️ to see the differences.  Please make sure that your mobile menu design is similar to the figma template  👍


####  Your design
mobbbbbb


#### Figma design
223529223-03b4d05c-fd1c-4b95-821c-ce6be91a4aa6

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @adeola003 @yin-ka , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linters errors ✔️ 
- Good use of github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (adeola003) [reply styles.css:34]: Thank you for your time @codecaiine. This is my last review so I wanted to be sure of what I need to change.
If I understand you well, I need to work on the following points:
- Move the button a little bit to the left;
- Increase the space between the menu items;
- Reduce the font size of the menu items;
Thanks, and please confirm If I'm right.
Best,

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @adeola003 :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**